docs/hardware/devices/seeed-studio/esp32-sx1262-kit/index.mdx

```diff
@@ -25,6 +30,12 @@ A thumb-sized LoRa dev kit, supporting WiFi, BLE, and LoRa, it includes a built-
 
 - **Thumb-sized Compact Design**: 21 x 18mm, adopting the classic form factor of XIAO.
 - **Grove Connectors**: 400+ Grove-compatible GPIOs for flexible expansion options.
+- **L76K GNSS Module available**: A Seeed L76K GNSS Module is available with the same form factor as the XIAO ESP32S3 & Wio-SX1262 Kit.
+
+:::info[Note]
+When using the Seeed L76K GNSS Module together with this kit, the L76K `RST` pin is shared with the Wio-SX1262 `RST` pin.
```

Comment:
I don't think this needs to be in an info admonition. Just adding it to the point above or detail page is going to be a better option